### PR TITLE
[feature](merge-on-write) enable merge-on-write by default

### PR DESCRIPTION
**Pull request by bobhan1**
## Proposed changes
1. make the default implementation to merge on write for unique tables.
2. add docs for table property `enable_unique_key_merge_on_write`.
3. display the property `enable_unique_key_merge_on_write` in `show create table` command for unique table regardless the value of the property.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by zhannngchen** [comment]
you can refer to my previous PR: #19017

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run p0

File: docs/zh-CN/docs/data-table/data-model.md [hunk 1]
```diff
@@ -416,7 +416,7 @@ mysql> select sum_merge(k2) , group_concat_merge(k3)from aggstate where k1 != 2;
 
 在某些多维分析场景下，用户更关注的是如何保证 Key 的唯一性，即如何获得 Primary Key 唯一性约束。
 因此，我们引入了 Unique 数据模型。在1.2版本之前，该模型本质上是聚合模型的一个特例，也是一种简化的表结构表示方式。
-由于聚合模型的实现方式是读时合并（merge on read)，因此在一些聚合查询上性能不佳（参考后续章节[聚合模型的局限性](#聚合模型的局限性)的描述），
+由于聚合模型的实现方式是读时合并(merge on read)，因此在一些聚合查询上性能不佳（参考后续章节[聚合模型的局限性](#聚合模型的局限性)的描述），
 在1.2版本我们引入了Unique模型新的实现方式，写时合并（merge on write），通过在写入时做一些额外的工作，实现了最优的查询性能。
 写时合并将在未来替换读时合并成为Unique模型的默认实现方式，两者将会短暂的共存一段时间。下面将对两种实现方式分别举例进行说明。
```
Comment (zhannngchen) [inline docs/zh-CN/docs/data-table/data-model.md:421]: “写时合并将在未来替换读时合并成为Unique模型的默认实现方式，两者将会短暂的共存一段时间”
update this description

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**